sentry-android-gradle-plugin/src/main/groovy/io/sentry/android/gradle/SentryPlugin.groovy

```diff
@@ -329,18 +334,44 @@ class SentryPlugin implements Plugin<Project> {
                         if (buildTypeProperties.has(SENTRY_ORG_PARAMETER)) {
                             args.add("--org")
                             args.add(buildTypeProperties.get(SENTRY_ORG_PARAMETER).toString())
+
+                            nativeArgs.add("-o")
+                            nativeArgs.add(buildTypeProperties.get(SENTRY_ORG_PARAMETER).toString())
                         }
                         if (buildTypeProperties.has(SENTRY_PROJECT_PARAMETER)) {
                             args.add("--project")
                             args.add(buildTypeProperties.get(SENTRY_PROJECT_PARAMETER).toString())
+
+                            nativeArgs.add("-p")
+                            nativeArgs.add(buildTypeProperties.get(SENTRY_PROJECT_PARAMETER).toString())
+                        }
+
+                        nativeArgs.add("${project.projectDir}/build")
+
+                        if (extension.includeNativeSource) {
+                            nativeArgs.add("--include-sources")
                         }
 
+                        project.logger.info("cli args: ${args.toString()}")
+
+                        project.logger.info("nativeArgs args: ${nativeArgs.toString()}")
+
                         if (Os.isFamily(Os.FAMILY_WINDOWS)) {
                             commandLine("cmd", "/c", *args)
                         } else {
                             commandLine(*args)
                         }
 
+                        project.logger.info("args executed.")
+
+                        if (Os.isFamily(Os.FAMILY_WINDOWS)) {
```

Comment:
Do we need some code here to figure out if we need to run the command or not?
For simplicity sake doesn't matter to build the command even if we're not running it, but before running I guess some auto detection of ndk build and perhaps also a configuration to opt-out.

Reply:
so there are 2 ways, one way is to add a field to SentryPluginExtension as I did with `includeNativeSource` and the 2nd is to find the NDK build task and hook into it, the 2nd is a bit more problematic because Gradle usually introduces breaking changes, but it doesn't require any additional setters on the client-side, also it requires more testing (Gradle and AGP versions).
I'm happy with both, but I'm afraid that I won't have the time to do it today.